frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -95,7 +95,11 @@ const TranslationPage = () => {
     );
   };
 
-  const onUserInput = async (newContent: string, lineIndex: number) => {
+  const onUserInput = async (
```

Comment:
is this a result of local linting out of sync with the docker image's linting?

Reply:
I had to manually lint this because it wouldn't compile. The reason why it's different from main is because there are more arguments in this function and the linter decided to push each parameter onto its own line.

Reply:
I had to manually lint this because it wouldn't compile. The reason why it's different from main is because there are more arguments in this function and the linter decided to push each parameter onto its own line.